Unity/Assets/YarnSpinner/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "com.thesecretlab.yarnspinner",
```

Comment:
Change package name to "dev.yarnspinner.yarnspinner"

Reply:
Done: ac9b44da31cfc95d01bc33ea284ef9135e09933b